databricks_cli/clusters/cli.py

```diff
@@ -331,6 +331,41 @@ def cluster_events_cli(api_client, cluster_id, start_time, end_time, order, even
         click.echo(tabulate(_cluster_events_to_table(events_json), tablefmt='plain'))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--cluster-id', cls=OneOfOption, one_of=CLUSTER_OPTIONS,
+              type=ClusterIdClickType(), default=None, help=ClusterIdClickType.help)
+@click.option('--cluster-name', cls=OneOfOption, one_of=CLUSTER_OPTIONS,
```

Comment:
Do you know if this is the standard way to define help for cluster name?  It's slightly confusing since it would indicate that users have to use cluster ID.

Reply:
yeah.. that's the standard help for cluster name on cluster cli. it's wrong, but we can try to fix it for all clusters cli on a separate PR.